client/src/modules/profiles/graphql/queries.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from '@apollo/client';
+
+export const userProfileQuery = gql`
+  query userProfile {
+    userInformation {
+      id
+      name
+      email
+      auto_subscribe
+      image_url
+      instance_role {
+        name
+      }
+      admined_chapters {
+        id
+        name
+      }
```

Comment:
for future, user_chapters have chapters in it, but the field doesn't allow for the client to request them, so the fields will be updated